code/api/src/config/database.json

```diff
@@ -1,18 +1,26 @@
 {
   "development": {
-    "username": "postgres",
+    "username": "carlyc",
```

Comment:
Should we configure all our environments so that we can all be using postgres

Reply:
This sounds like we would want to configure this to use Environment variables.